cypress/e2e/upload-download.cy.ts

```diff
@@ -0,0 +1,16 @@
+import {UploadPage} from "../page/index";
+
+describe("Upload/Download Test", () =>{
+  let upload: UploadPage;
+  const assertMsg = "File Uploaded!";
+
+  before(() => {
+    upload = new UploadPage();
+  });
+
+  it("Uploading a File", () => {
+    upload.visitUploadPage();
+    upload.uploadFile();
+    upload.assertMsg(assertMsg);
```

Comment:
the assertion gives us more value if we verify that the file that was intended to upload was effectively uploaded, the upload page shows the file name that was loaded, so, probably this method will be better if it verifies that the page shows the same file name used in the uploadFile method.

To be more clear, I expect this test class to have:

1. The file name that will be uploaded
2. The uploadFile method use that file name to know which file needs to be loaded
3. The verification or assertion method to use that same file name to verify that file was loaded correctly